docs/src/introduction.md

```diff
@@ -16,6 +16,8 @@ The simplified version of the language is based on [this discussion](https://git
 
 The language will be specified in detail in the rest of this book.
 
+The code for AirScript can be found [here](https://github.com/0xPolygonMiden/air-script/).
+
```

Comment:
It might be good to put the link right at the top so it's easiest to find. Maybe we could either edit the first sentence to include a link or add the sentence right after it?

Polygon Miden's [AirScript](https://github.com/0xPolygonMiden/air-script/) is designed to make it simple to describe AIR constraints and generate efficient and accurate code in the required target language.

OR

Polygon Miden's AirScript is designed to make it simple to describe AIR constraints and generate efficient and accurate code in the required target language. The code for AirScript can be found [here](https://github.com/0xPolygonMiden/air-script/).

Reply:
I went with the second option because it might be easy to overlook the link in the first.

Reply:
Another, potentially complementary, option is to go with the [corresponding option in mdbook](https://rust-lang.github.io/mdBook/format/configuration/renderers.html#html-renderer-options). In `book.toml`:

```toml
[output.html]
git-repository-url = "https://github.com/0xPolygonMiden/air-script/"
```

Reply:
@jan-ferdinand Thanks. This looks great.